examples/VLCB_SerialGC_1in1out/VLCB_SerialGC_1in1out.ino

```diff
@@ -0,0 +1,278 @@
+
+// constants
+const byte VER_MAJ = 1;             // code major version
+const char VER_MIN = 'a';           // code minor version
+const byte VER_BETA = 0;            // code beta sub-version
+const byte MANUFACTURER = MANU_DEV; // for boards in development.
+const byte MODULE_ID = 99;          // VLCB module type
+
+const byte LED_GRN = 4;             // VLCB green Unitialised LED pin
+const byte LED_YLW = 7;             // VLCB yellow Normal LED pin
+const byte SWITCH0 = 8;             // VLCB push button switch pin
+
+// Controller objects
+VLCB::Configuration modconfig;               // configuration object
+//VLCB::CAN2515 can2515;                  // CAN transport object
+VLCB::SerialGC serialGC;                  // CAN transport object using serial
+VLCB::LEDUserInterface ledUserInterface(LED_GRN, LED_YLW, SWITCH0);
+//VLCB::SerialUserInterface serialUserInterface(&can2515);
+VLCB::MinimumNodeService mnService;
+VLCB::CanService canService(&serialGC);
+VLCB::NodeVariableService nvService;
+VLCB::ConsumeOwnEventsService coeService;
+VLCB::EventConsumerService ecService;
+VLCB::EventTeachingService etService;
+VLCB::EventProducerService epService;
+VLCB::Controller controller(&modconfig, 
+                            {&mnService, &ledUserInterface, &canService, &nvService, &ecService, &epService, &etService, &coeService}); // Controller object
+/*
+VLCB::Controller controller(&modconfig, 
+                            {&mnService, &ledUserInterface, &serialUserInterface, &canService, &nvService, &ecService, &epService, &etService, &coeService}); // Controller object
+*/
+
+// module objects
+VLCB::Switch moduleSwitch(5);            // an example switch as input
+VLCB::LED moduleLED(6);                  // an example LED as output
+
+// module name, must be 7 characters, space padded.
+unsigned char mname[7] = { '1', 'I', 'N', '1', 'O', 'U', 'T' };
+
+// forward function declarations
+void eventhandler(byte, const VLCB::VlcbMessage *);
+void printConfig();
+void processModuleSwitchChange();
+
+//
+/// setup VLCB - runs once at power on from setup()
+//
+void setupVLCB()
+{
+  // set config layout parameters
+  modconfig.EE_NVS_START = 10;
+  modconfig.EE_NUM_NVS = 10;
+  modconfig.EE_EVENTS_START = 20;
+  modconfig.EE_MAX_EVENTS = 32;
+  modconfig.EE_PRODUCED_EVENTS = 1;
+  modconfig.EE_NUM_EVS = 2; // EV1: Produced event ; EV2: LED1
+ 
+
+  // initialise and load configuration
+  controller.begin();
+
+  const char * modeString;
+  switch (modconfig.currentMode)
+  {
+    case MODE_NORMAL: modeString = "Normal"; break;
+    case MODE_SETUP: modeString = "Setup"; break;
+    case MODE_UNINITIALISED: modeString = "Uninitialised"; break;
+    default: modeString = "Unknown"; break;
+  }
+  Serial << F("> mode = (") << _HEX(modconfig.currentMode) << ") " << modeString;
+  Serial << F(", CANID = ") << modconfig.CANID;
+  Serial << F(", NN = ") << modconfig.nodeNum << endl;
+
+  // show code version and copyright notice
+  printConfig();
```

Comment:
Does this serial output upset the GridConnect communication?
If it does, remove everything from line 95 to here.